src/south/FolderScanner/FolderScanner.schema.js

```diff
@@ -1,21 +1,25 @@
-import React from 'react'
-import { notEmpty, minValue } from '../../services/validation.service'
-
-const schema = { name: 'FolderScanner' }
+const schema = {
+  name: 'FolderScanner',
+  category: 'FileOut',
+  supportListen: false,
+  supportLastPoint: false,
+  supportFiles: true,
+  supportHistory: false,
+}
 schema.form = {
   FolderScannerSettings: {
     type: 'OIbTitle',
     label: 'FolderScanner settings',
     md: 12,
-    children: (
-      <>
+    children: `
+      <div>
         <p>
           The FolderScanner South periodically checks the input folder for new files at an interval specified by scan mode.
           When a new file is detected it will be sent to any North capable of handling files and configured to accept files from this South.
         </p>
         <p>
           Please use always the / separator. OIBus will convert it according to the platform.
-          For example, on Windows, c:/input/ will be converted to c:\input\
+          For example, on Windows, c:/input/ will be converted to c:\\input\\
```

Comment:
I think we don't need the last / anymore

Reply:
Also, I think the way we now parse the path, it does not matter if it's windows or linux style